pkg/segment/reader/record/rrcreader_test.go

```diff
@@ -245,3 +245,43 @@ func Test_GetJsonFromAllRrc_withAggs_ExcludeCols(t *testing.T) {
 
 	os.RemoveAll(dir)
 }
+
+//Variables are defined below
+type NodeResult struct{}
```

Comment:
Why do you need to make a new type for this unit test? Can't you just use the existing `NodeResult` type from segstructs.go?